src/app/drive/components/ShareDialog/ShareDialog.tsx

```diff
@@ -93,6 +93,7 @@ type ShareDialogProps = {
   user: UserSettings;
   isDriveItem?: boolean;
   onShareItem?: () => void;
+  onCloseDialog?: () => void;
 };
 
 const isAdvanchedShareItem = (item: DriveItemData | AdvancedSharedItem): item is AdvancedSharedItem => {
```

Comment:
Im a bit confused by this, a `DriveItemData` can be a shared item, but a shared item is an `AdvancedSharedItem` which is not the same as a `DriveItemData`, but could be 😅

Reply:
In principle an `AdvancedSharedItem` is like `DriveItemData` with some extra fields related to the new file sharing, I'll check it in case I introduce some changes in the types.